### ISSUE TITLE
Other issues found

### ISSUE THREAD

**Issue by treviser** [lukebranch/odoo-install-scripts#4]
http://kaz.dl.sourceforge.net/project/wkhtmltopdf/0.12.1/wkhtmltox-0.12.1_linux-trusty-amd64.deb

should be: http://kaz.dl.sourceforge.net/project/wkhtmltopdf/0.12.2.1/wkhtmltox-0.12.2.1_linux-trusty-amd64.deb

sudo easy_install uno -y
sudo easy_install -U distribute -y

easy_install doesn't seem to have an option -y, but I don't know what you want to achieve. It seems to work anyway, but it's not nice :-)

sudo pip install graphviz ghostscript gcc mc bzr lptools make

There is no gcc package available, however Ubuntu 14.04 comes with gcc-4.8 preinstalled, so maybe it is not necessary.

**Comment by cliffkujala** [comment]
Are you sure about using wkhtmltopdf 0.12.2.1?  When I tried it it broke the print layouts for me. Switching back to 0.12.1, fixed all again.

**Comment by treviser** [comment]
No, I'm not sure, but in that case the link should be http://sourceforge.net/projects/wkhtmltopdf/files/archive/0.12.1/wkhtmltox-0.12.1_linux-wheezy-amd64.deb.
0.12.2.1 would have the dependencies on the packages xfonts-base and xfonts-75dpi.
